src/abstractdataframe/abstractdataframe.jl

```diff
@@ -1903,7 +1903,8 @@ const hashdf_seed = UInt == UInt32 ? 0xfd8bb02e : 0x6215bada8c8c46de
 
 function Base.hash(df::AbstractDataFrame, h::UInt)
     h += hashdf_seed
```

Comment:
Apparently https://github.com/JuliaLang/julia/pull/57509 switched all similar cases from `+` to `⊻`, so I assume there's a reason (and better be consistent with what you do below).
```suggestion
    h ⊻= hashdf_seed
```